var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -540,12 +540,10 @@ def cmake_args(self):
         if spec.satisfies("@8.5.0:"):
             options.append(self.define_from_variant("DEAL_II_COMPONENT_PYTHON_BINDINGS", "python"))
             if "+python" in spec:
-                python_exe = spec["python"].command.path
                 python_library = spec["python"].libs[0]
                 python_include = spec["python"].headers.directories[0]
                 options.extend(
                     [
-                        self.define("PYTHON_EXECUTABLE", python_exe),
                         self.define("PYTHON_INCLUDE_DIR", python_include),
                         self.define("PYTHON_LIBRARY", python_library),
```

Comment:
Are these still needed? See `clingo` above where you removed them.

Reply:
Maybe not. I am currently cleaning up the last patch and will come back around to these and test them. There are a few failures in CI and a few more failures that didn't come up in CI but are issues that need to be resolved.

Reply:
Yeah, these extra hints are probably unnecessary. Others that determine the install dir could probably stay.

Reply:
Actually it depends. If `find_package(PythonLibs)` is called before `find_package(PythonInterp)` we need to set those.